build_tools/update_environments_and_lock_files.py

```diff
@@ -618,6 +626,13 @@ def main(verbose, very_verbose, select_build):
         for each in conda_build_metadata_list
         if re.search(select_build, each["build_name"])
     ]
+    if skip_build is not None:
+        filtered_conda_build_metadata_list = [
+            each
+            for each in filtered_conda_build_metadata_list
+            if not re.search(skip_build, each["build_name"])
+        ]
```

Comment:
`filtered_pip_build_metadata_list` should be filtered the same way below.

Reply:
Done